eth/tracers/api_blocktrace.go

```diff
@@ -211,6 +213,13 @@ func (api *API) getBlockTrace(block *types.Block, env *traceEnv) (*types.BlockTr
 		}
 	}
 
+	// build dummy per-tx deletion proof
+	for _, txs := range env.txStorageTrace {
```

Comment:
```suggestion
	for _, trace := range env.txStorageTrace {
```

Reply:
fixed in https://github.com/scroll-tech/go-ethereum/pull/372/commits/7b92a484fe79dcae22e697cdfdab742f480acefb